### PR TITLE
More indentation for test description.

### PR DESCRIPTION
**Pull request by garloff**
Otherwise, a contradiction could be read into the wording.

This is a tiny improvement on top of #700.

### CODE REVIEW

**Comment by mbuechse** [comment]
@garloff  I pushed an alternative proposal

```text
wavestack SCS-compatible IaaS v4 (effective) [tests: 'standard-flavors-check']:
- main: FAIL (0 passed, 1 failed, 4 missing)
  - FAILED:
    - standard-flavors-check:
      > The environment fulfills all requirements of https://docs.scs.community/standards/scs-0103-v1-standard-flavors
  - MISSING:
    - flavor-name-check:
      > The environment fulfills all requirements of https://docs.scs.community/standards/scs-0100-v3-flavor-naming
    - entropy-check:
      > The environment fulfills all requirements of https://docs.scs.community/standards/scs-0101-v1-entropy
    - image-metadata-check:
      > The environment fulfills all requirements of https://docs.scs.community/standards/scs-0102-v1-image-metadata
    - standard-images-check:
      > The environment fulfills all requirements of https://docs.scs.community/standards/scs-0104-v1-standard-images
```

This is an example where certain test cases are missing (imagine the respective script just crashed).

The categories Missing and Failed have different lengths. The additional indentation that you proposed has two disadvantages:

- either it must be aligned with the length of the category, but that looks weird, or we don't align it, and that looks weird as well
- it takes up even more space

**Comment by garloff** [comment]
OK, this is better than the original.

File: Tests/scs-compatible-iaas.yaml [hunk 1]
```diff
@@ -152,7 +152,7 @@ modules:
     testcases:
       - id: standard-images-check
         tags: [mandatory]
-        description: |
+        description: >
           The environment fulfills all requirements of https://docs.scs.community/standards/scs-0104-v1-standard-images
```
Comment (garloff) [inline Tests/scs-compatible-iaas.yaml:156]:
```suggestion
          The environment should fulfill all requirements of https://docs.scs.community/standards/scs-0104-v1-standard-images
```
Maybe the wording with "should fulfill" is better. As it is unknown and depends on the result of the check.
This also avoid any perceived contradiction between `FAILED` and `fulfills`.

Comment (garloff) [reply Tests/scs-compatible-iaas.yaml:156]: Or maybe "Passing indicates that the environment fulfills all requirements ..."

Comment (mbuechse) [reply Tests/scs-compatible-iaas.yaml:156]:
```suggestion
          Must fulfill all requirements of https://docs.scs.community/standards/scs-0104-v1-standard-images
```

Comment (garloff) [reply Tests/scs-compatible-iaas.yaml:156]: I like this one the best thus far.

**Review by mbuechse — APPROVED** [review]

**Pull request merged**